cloud-functions/http_standardize_names/test_http_standardize_names.py

```diff
@@ -0,0 +1,81 @@
+import json
+from main import http_standardize_names
+from unittest import mock
+import pytest
+
+
+def test_standardize_names_bad_header():
+    request = mock.Mock(headers={"Content-Type": "not-application/json"})
+
+    assert http_standardize_names(request) == {
+        "status": 400,
+        "summary": "Bad request",
+        "description": "Header must inclue: 'Content-Type:application/json'.",
+    }
+
+
+def test_standardize_names_bad_body():
+    request = mock.Mock(headers={"Content-Type": "application/json"})
+    request.get_json.return_value = ""
+    with pytest.raises(BaseException):
+        http_standardize_names(request=request)
+
```

Comment:
If you take my suggestion above we should add an additional request body test here, or expand the existing one.

Reply:
I will add the test for sure.  This makes sense.